src/plugins/Transak/hooks/useTransak.ts

```diff
@@ -0,0 +1,26 @@
+import TransakSDK, { TransakSDKConfig } from '@transak/transak-sdk'
+import stringify from 'json-stable-stringify'
+import { useMemo } from 'react'
+
+export function useTransak(config?: Partial<TransakSDKConfig>) {
+    const transak = useMemo(
+        () =>
+            new TransakSDK({
```

Comment:
Does it need to "close" the SDK if config changes?

Reply:
https://github.com/DimensionDev/Maskbook/pull/1732/files#diff-9153e7f2a7688ef203ed9bb35ea00fa16eea77cad5ec0f501251e41c1c4d155eR30